### PR TITLE
Forward all mailgun mail to alpha. subdomains

### PR DESCRIPTION
**Pull request by Matthijsy**
### Summary
This is a bit tricky PR. It will forward all mail from mailgun to the same alias but on the `alpha` subdomain, which is configured on our new mailprovider. The tricky part is that I cannot test it locally, since we cannot setup mailgun anymore to a local system. I hope it will work!

### CODE REVIEW

File: spec/jobs/mail_forward_job_spec.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
     it do
       expect(fake_http).to have_received(:post).with(
-        message_url, form: { to: mail_alias.user.email }
+        message_url, form: { to: 'testing@alpha.csvalpha.nl' }
```
Comment (guidojw) [inline spec/jobs/mail_forward_job_spec.rb:21]: Isn't the old way here still possible? Like reading the email directly from mail_alias

Comment (Matthijsy) [reply spec/jobs/mail_forward_job_spec.rb:21]: No, because the mail_alias in this case is `testing@csvalpha.nl` and we do forward to the `alpha` subdomain

**Review by guidojw — APPROVED** [review]
Looks good to me

**Comment by Matthijsy** [comment]
FYI: Do not merge at the moment!

**Comment by Matthijsy** [comment]
Open aliases are running fine for more than a week now. I think we can start testing the closed aliases as well. What do you guys think? @wilco375 @guidojw @cpbscholten

File: app/jobs/mail_receive_job.rb [hunk 2]
```diff
@@ -3,34 +3,39 @@
...
-  end
 
-  def send_mail_moderations(mail_alias, message_url, fetched_mail)
-    stored_mail = StoredMail.create(message_url: message_url, received_at: fetched_mail.received_at,
-                                    sender: fetched_mail.sender, mail_alias: mail_alias,
-                                    subject: fetched_mail.subject)
-
-    mail_alias.moderators.each do |moderator|
-      MailModerationMailer.request_for_moderation_email(moderator, stored_mail).deliver_later
+      # if mail_alias.moderation_type == 'open'
```
Comment (wilco375) [inline app/jobs/mail_receive_job.rb:15]: I think this can be removed, right? That's why we use Git :)

File: app/jobs/mail_receive_job.rb [hunk 3]
```diff
@@ -3,34 +3,39 @@
...
+      # else
+      #   send_mail_moderations(mail_alias, message_url, fetched_mail)
+      # end
+      #
     end
-    MailModerationMailer.awaiting_moderation_email(stored_mail.sender, stored_mail).deliver_later
-    MailModerationReminderJob.set(wait: 24.hours).perform_later(stored_mail.id)
   end
 
+  # def send_mail_moderations(mail_alias, message_url, fetched_mail)
```
Comment (wilco375) [inline app/jobs/mail_receive_job.rb:26]: Same for this

**Review by wilco375 — CHANGES_REQUESTED** [review]

**Comment by wilco375** [comment]
Sounds good. Just some small complains about the code 😅

File: app/jobs/mail_receive_job.rb (hunk 2 above)
Comment (Matthijsy) [reply app/jobs/mail_receive_job.rb]: Good point :)

**Review by wilco375 — APPROVED** [review]

**Pull request merged**